assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -49,13 +49,17 @@ function DropTargetsProvider({ children }) {
   const getDropTargetFromCursor = useCallback(
     (x, y, ignoreId = null) => {
       const underCursor = document.elementsFromPoint(x, y);
+      const sortedDropTargetIds = (currentPage?.elements || [])
```

Comment:
It looks like `getDropTargetFromCursor` is called a lot? Would it be better to do:

```
const sortedDropTargetIds = useMemo(() => ..., [dropTargets, currentPage?.element]);

const getDropTargetsFromCursor = useCallback( () => ..., [sortedDropTargetIds])
```
?